src/ui/components/inputs/InputError.tsx

```diff
@@ -0,0 +1,34 @@
+import { t } from '@lingui/macro'
+import React, { FC, Fragment } from 'react'
+import styled from 'styled-components/native'
+
+import { _ } from 'libs/i18n'
+import { Warning } from 'ui/svg/icons/Warning'
+import { ColorsEnum, getSpacing, Spacer, Typo } from 'ui/theme'
+
+interface Props {
+  messageId: string
+  visible: boolean
+  numberOfSpacesTop: number
```

Comment:
Je suis pas sûr pour cette prop. Un avis @Giregm ?

Reply:
Si cela ne gêne personne autre que moi de gérer l'espace de cette manière, alors why not.

Reply:
J'ai le sentiment que ces 3 façons d'écrire rendent pas loin de la même chose (c'est une première impression):

```typescript
<InputError visible={visible} numberOfSpacesTop={3} />
```
et
```typescript
{visible && <Spacer.Column numberOfSpaces={3}/>}
<InputError visible={visible}  />
```
et
```typescript
{visible && (
  <Spacer.Column numberOfSpaces={3} />
  <InputError />
)}
```

Peut être qu'une des 2 dernières façon est plus simple, qu'en penses-tu Alain ?